spec/models/manageiq/providers/microsoft/infra_manager/refresher_spec.rb

```diff
@@ -115,15 +115,15 @@ def assert_specific_cluster
   def assert_specific_host
     @host = ManageIQ::Providers::Microsoft::InfraManager::Host.find_by_name("hyperv-h01.manageiq.com")
     expect(@host).to have_attributes(
-      :ems_ref          => "60e92646-b9f8-432a-a71a-5bc169ceeca2",
-      :name             => "hyperv-h01.manageiq.com",
-      :hostname         => "hyperv-h01.manageiq.com",
-      :ipaddress        => "192.168.252.99",
-      :vmm_vendor       => "Microsoft",
-      :vmm_version      => "6.3.9600.17039",
-      :vmm_product      => "HyperV",
-      :power_state      => "on",
-      :connection_state => "connected"
+      :ems_ref            => "60e92646-b9f8-432a-a71a-5bc169ceeca2",
+      :name               => "hyperv-h01.manageiq.com",
+      :hostname           => "hyperv-h01.manageiq.com",
+      :ipaddress          => "192.168.252.99",
+      :vmm_vendor_display => "Microsoft",
```

Comment:
Refresher specs shouldn't care about `vmm_vendor_display` either, since they are not setting them.   They should only care about `vmm_vendor`